src/datachain/lib/dc.py

```diff
@@ -325,6 +325,7 @@ def settings(
         parallel=None,
         workers=None,
         min_task_size=None,
+        prefetch: Optional[int] = None,
```

Comment:
q: why int? let's update the docs here (do we have some CI to detect these discrepancies btw (missing docs) cc @skshetry )